src/gui/components/navbar/navbar.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import container, { title } from './navbar.styles'
+
 const Navbar = ({ style, className }) => {
   return (
-    <div style={style} className={className}>
-      Workbench
+    <div style={style} className={`${container} ${className}`}>
```

Comment:
Because `container` is a glamor object, we can use `classnames` but we need to use it like that : 
```javascript
classnames(`${container}`, className)
classnames(container.toString(), className)
```
But with the `css` method of glamor, we could declare css classes like that :
```javascript
<div className={css(container, className)} />
```
FYI, other way to declare className : `<div {...container} />`